### PR TITLE
Feature ests 137052 2

### PR DESCRIPTION
**Pull request by madhas8**
- [ ] I have read the [CONTRIBUTION GUIDE][contributing]

- [ ] This is a bug

- [ ] This is a feature request

- [ ] This is a syntax/style fix

- [ ] This is an improvement to code quality

---
**Please describe your changes below:**
(Note: A clear description of changes is required to approve any pull requests)

_"Changed the implementation for verification because..."_

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/MessageAnnotationProcessor.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 public class MessageAnnotationProcessor
 {
     private static final String DEFAULT_PACKAGE_TO_SCAN = "com.dell.cpsd";
-    private Logger                      LOG                     = LoggerFactory.getLogger(MessageAnnotationProcessor.class);
+    private static final Logger log                     = LoggerFactory.getLogger(MessageAnnotationProcessor.class);
```
Comment (ganapn) [inline common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/MessageAnnotationProcessor.java]: LOG is better than log as it is a final variable.

File: pom.xml [hunk 2]
```diff
@@ -99,6 +103,12 @@
             <organization>com.dell.cpsd</organization>
             <organizationUrl>https://github.com/joz73</organizationUrl>
         </developer>
+        <developer>
```
Comment (ganapn) [inline pom.xml]: Add all the developer names from our team except me :)

Comment (madhas8) [reply pom.xml]: Developer names addition is part of Open source story :)

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/MessageAnnotationProcessor.java (hunk 1 above)
Comment (madhas8) [reply common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/MessageAnnotationProcessor.java]: Done

**Pull request merged**